db/model/model.js

```diff
@@ -128,7 +129,39 @@ exports.deleteCommentById = (comments_id) => {
 };
 
 exports.getAllUsers = () => {
-  return db.query(`SELECT * FROM users`).then(({ rows }) => {
+  let queryStr = `SELECT * FROM users`;
+
+  return db.query(queryStr).then(({ rows }) => {
+    return rows;
+  });
+};
+
+exports.orderArticles = (sort_by, order_by) => {
+  const validInputs = [
+    "article_id",
+    "title",
+    "topic",
+    "author",
+    "body",
+    "created_at",
+    "votes",
+    "article_img_url",
+  ];
+  console.log(sort_by);
+  if (!sort_by || !validInputs.includes(sort_by)) {
```

Comment:
There's no requirement to check if sort_by is defined here. As per the task instructions, it should have a default value of "created_at" so would always be defined. Please ensure you include this 👍